api/Navigator.json

```diff
@@ -281,10 +283,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/buildID",
           "support": {
             "chrome": {
-              "version_added": null
```

Comment:
Do these `buildID` changes have background info or are they actually battery-related in the wrong place?

Reply:
This is based on the fact that the property is not in Chrome. You can open Chrome DevTools and type `navigator.` in the console. You'll see that buildID is missing from the list. I've also confirmed that it's not in Chrome's source code.